lib/livebook/runtime/evaluator.ex

```diff
@@ -999,6 +1156,8 @@ defmodule Livebook.Runtime.Evaluator do
   @doc false
   def write_module!(module, bytecode) do
     if ebin_path = ebin_path() do
+      File.mkdir_p!(Path.dirname(ebin_path))
```

Comment:
Did you run into any failure without this? We generally create this directory upfront.

Reply:
I had an intermediate code which failed. But I agree - this should not be needed. Will remove.